### PR TITLE
json parser created

### PR DESCRIPTION
**Pull request by kailamink**

### CODE REVIEW

File: src/main/java/pinball/BodyType.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package pinball;
+
+public enum BodyType {
+    BOTTOM_WALL("BOTTOM_WALL"),
```
Comment (andrewoid) [inline src/main/java/pinball/BodyType.java]: What's the difference between all these wall types?

File: src/main/java/pinball/PinballComponent.java [hunk 2]
```diff
@@ -1,16 +1,249 @@
...
+                bodies.add(createBody(bodyDTO));
+            }
+        }
+        catch (IOException e)
+        {
+            e.printStackTrace();
+        }
+
+        //set up all horizontal and vertical boxes
+        bottom = createWall(new float[] {100f, (HEIGHT + 100) },
```
Comment (andrewoid) [inline src/main/java/pinball/PinballComponent.java]: Why are you creating walls here as well as reading in the bodies from the json?

File: src/main/java/pinball/BodyType.java (hunk 1 above)
Comment (kailamink) [reply src/main/java/pinball/BodyType.java]: Makes a difference when drawing them

File: src/main/java/pinball/PinballComponent.java (hunk 2 above)
Comment (kailamink) [reply src/main/java/pinball/PinballComponent.java]: Ah good catch. Seems I hadn't pushed the latest

File: src/main/java/pinball/BodyType.java (hunk 1 above)
Comment (andrewoid) [reply src/main/java/pinball/BodyType.java]: It doesn't.

File: src/main/java/pinball/DTO/BodiesDTO.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package pinball.DTO;
+
+public class BodiesDTO {
+    public BodyDTO[] bodies;
```
Comment (andrewoid) [inline src/main/java/pinball/DTO/BodiesDTO.java]: public variables are not allowed

File: src/main/java/pinball/DTO/BodyDTO.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package pinball.DTO;
+
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+import pinball.BodyType;
+import pinball.ShapeType;
+
+public class BodyDTO {
+    public BodyType bodyType;
```
Comment (andrewoid) [inline src/main/java/pinball/DTO/BodyDTO.java]: public variables are not allowed

**Review by andrewoid — APPROVED** [review]

**Pull request merged**